modules/ols-enabling-cluster-inspection.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-cluster-inspection_{context}"]
+= Enabling cluster inspection 
+
+Modify the `OLSConfig` custom resource definition file to enable the cluster inspection feature. Enabling the feature allows you to enhance the large language model (LLM) so that it includes information about an {ocp-product-title} cluster.
+
+[NOTE]
+====
+Cluster inspection is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
+
+* You have configured the large language model (LLM) provider.
+
+* You have installed the {ols-long} Operator.
+
+.Procedure 
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*. 
+
+. Click {ols-long} Operator.
+
+. Click *OLSConfig*, then click the `cluster` configuration instance in the list.
+
+. Click the *YAML* tab.
+
+. Set the `spec.ols.introspectionEnabled` parameter to `true` to enable cluster inspection: 
++
+.Example `OLSconfig` CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+spec:
+  ols:
+    introspectionEnabled: true  
+----
+
+. Click *Save*.
+
+. Access the {ols-long} virtual assistant and submit a question associated with the custom content that was added to the LLM.
++
+The {ols-long} virtual assistant generates an answer based on the custom content.
```

Comment:
For consideration `The {ols-long} virtual assistant can use cluster information to generate an answer.`

Reply:
Thank you!

Done.